### DISCUSSION TITLE
Error trying to install DAT on new Intel PC w/Nvidia graphics

### DISCUSSION THREAD

**Discussion by TaoDoggy** [dat-linux/community#58] [Issues]
When I try to install DAT Linux on my computer, I get:
1.427844 nouveau 0000:01:00.0: unknown chipset (1920000a1)
then the system freezes.
System specs: 

CPU: Intel(R) Core™ Processor i9-14900KF 8P/16 + 16E 3.20GHz [Turbo 6.0GHz] 36MB Cache LGA1700 [w/o Integrated Graphic] 
VIDEO: GeForce RTX™ 4090 24GB GDDR6X Video Card (DLSS 3.0) [AI-Powered Graphics] 
IMEMORY: 64GB (32GBx2) DDR5-6000MHz MEMORY 
HDD: 2TB PCIe NVMe M.2 SSD 
MOTHERBOARD: Z790 DDR5 With WiFi Motherboard 
POWERSUPPLY: 1,000 Watts - Standard 80 Plus Gold Power Supply 
SUMMARY: Intel IET99613

**Comment by dat-linux** [comment]
Hi, at what point do you see this error message?

**Reply by TaoDoggy** [reply]
I can install the OS using the "safe graphics" mode, but when I boot into the installed system, this line appears.  I've been trying to edit grub, but I can't find it.  At the grub menu, I type 'c' and look for it with variations on "ls (hd0,1)/etc/default/grub with no luck.  Also, is there a way to install Nvidia proprietary drivers during the installation?

**Reply by TaoDoggy** [reply]
I found grub, but I can't find the editor.  vi doesn't work.

**Reply by TaoDoggy** [reply]
I've edited grub with nouveau.modeset=0 acpi=off and snd_hda_enable=0, ran update-grub.  When I rebooted the installed system, it was still hanging  after the message
6.678025] snd_hda_intel 0000:01:00.1: unable to grab IRQ 0, disabling device.  So I went back into the live media and blacklisted snd_hda_intel in etc/modprobe.d/blacklist.conf
update-initramfs -u have a bunch of "no such file" errors.  I tried to boot anyway and now it hangs after: 
[ 5.787779] systemd-journald [235]: Received client request to flush runtime journal.
[ OK] Finished Terminate Plymouth Boot Screen.

**Reply by TaoDoggy** [reply]
Ok.  The list of things I've tried is getting very long.  I can install Pop!_OS or Ubuntu Studio with no problems.  I think the current DAT kernel isn't compatible with my Intel CPU:  Intel(R) Core™ Processor i9-14900KF 8P/16 + 16E 3.20GHz [Turbo 6.0GHz] 36MB Cache LGA1700 [w/o Integrated Graphic].  The lack of Intel graphics and sound processors means the system needs the Nvidia drivers loaded.

**Reply by TaoDoggy** [reply]
At boot I now get:
0.754148] genirq: Flags mismatch irq 0.00000080 (1801_sabus) vs. 00015800

(timer)

[0.754152] 1801 sabus 0000:00:1f.4: Failed to allocate irq 0:-16 /dev/nvme0n1p2: clean, 429895/57606144 files, 6691932/230399999 blocks

My grub looks like this:
GRUB_CMDLINE_LINUX_DEFAULT="nouveau.modeset=0 text rhgb quiet plymouth.enable=0 nvidia-drm.modeset=1”

**Reply by TaoDoggy** [reply]
And yes, update-grub and update-initramfs -u complete without errors under chroot

**Reply by TaoDoggy** [reply]
And I can only use the live media in "safe graphics" mode.

**Reply by TaoDoggy** [reply]
I have Pro installed. If I change the desktop environment, will it mess up any of the features of DAT?  I'm thinking KDE...

**Comment by dat-linux** [comment]
DAT Linux is based off of Lubuntu 22.04, with updates as of Feb 21. So the ISO will contain Kernel 6.5.0-25. 

>Also, is there a way to install Nvidia proprietary drivers during the installation?

Nope. Can you get to a command line? Try this: https://askubuntu.com/questions/859630/how-to-start-ubuntu-in-console-mode

Then try installing the drivers and reverse the grub change.

`sudo apt update && sudo apt upgrade && sudo apt dist-upgrade `

 to get the latest kernel.

**Reply by TaoDoggy** [reply]
apt update, etc fail because the DNS resolver isn't working under chroot.
I can't boot into the installed OS, only to the live media.  To make changes to the installation, I have to run in chroot.

**Reply by TaoDoggy** [reply]
Can you help me with the DNS issue?

**Reply by TaoDoggy** [reply]
I tried booting the installation with text and consoles, but it still hangs.

**Reply by TaoDoggy** [reply]
Finally fixed it.  After looking at the logs, I saw it was having trouble assigning IRQs.  I added pci=biosirq to grub and it was able to boot.

**Reply by TaoDoggy** [reply]
I have Pro installed.  If I change the desktop environment will it mess up any of the fine features of DAT?  I'm thinking KDE....
